packages/nx/src/project-graph/build-project-graph.ts

```diff
@@ -235,15 +234,8 @@ async function updateProjectGraphWithPlugins(
     try {
       graph = await plugin.processProjectGraph(graph, context);
     } catch (e) {
-      const message = `Failed to process the project graph with "${plugin.name}". This will error in the future!`;
-      if (process.env.NX_VERBOSE_LOGGING === 'true') {
-        console.error(e);
-        logger.error(message);
-        return graph;
-      } else {
-        logger.warn(message);
-        logger.warn(`Run with NX_VERBOSE_LOGGING=true to see the error.`);
-      }
+      const message = `Failed to process the project graph with "${plugin.name}".`;
```

Comment:
Do you think we should just throw the real error? People can see from the stacktrace where it is from right?

Reply:
I like having this line in here - the stack trace may show something like "Unable to read property X from undefined" and I can see folks opening that issue up here when its from a comm. plugin. 

Putting the plugin name front and center should make it more likely that those issues don't start on our repo. I do think we should show the e.message too though - I'll get that updated.